src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import java.util.List;
+
+/**
+ * Sorts the address book
```

Comment:
The header comment should provide the usage of the class, in this case can we be more detailed and precise with what this class is supposed to handle =)? Is the class supposed to `Sorts persons in the address book, according to their names in lexicographical order`?

Reply:
Yup